### ISSUE TITLE
Build fails under the latest release of Bazel (0.27.0)

### ISSUE THREAD

**Issue by superphiz** [prysmaticlabs/prysm#2830]
When building target //validator:validator with the latest current version of bazel (0.27.0) the following error is encountered. Regressing to bazel 0.26.0 resolves the error.

ERROR... 'single_file' is no longer supported. use allow_single_file instead. You can use --incompatible_disable_deprecated_attr_params=false to temporarily disable this check.

**Comment by prestonvanloon** [comment]
Duplicate of #2822. We'll need to merge #2822 to resolve this. I'll ping for code review on it.

**Comment by prestonvanloon** [comment]
@superphiz #2823 is merged now and you should be unblocked here.

Sorry about that!

**Comment by eliotstock** [comment]
I'm still seeing this error when building master using bazel 0.27.1. I'm trying to build commit f12fdfda0f2e4e27230422209fab7b092bb75bae.

**Comment by prestonvanloon** [comment]
@eliotstock It should be resolved again now! Thanks for reporting :)
